src/helpers/portfolioHelper.ts

```diff
@@ -224,7 +224,7 @@ export const getPortfoliosByUser = async (
   // also join securities
   const prisma = new PrismaClient();
 
-  const portfolios = await prisma.portfolio.findMany({
+  const portfolios : PortfolioJoined[] = await prisma.portfolio.findMany({
```

Comment:
this isn't going to be joined yet (see type error in `npm run build`) -- the prisma output doesn't return the joined types, we calculate some of that internally

Reply:
will remove then